app/src/ui/app-menu/app-menu.tsx

```diff
@@ -333,7 +333,6 @@ export class AppMenu extends React.Component<IAppMenuProps, {}> {
     if (this.focusPane >= 0) {
       const pane = this.paneRefs[this.focusPane]
       if (pane) {
-        pane.focus()
```

Comment:
@niik Do you remember context around this pane focusing? Currently, it is problematic and removing "fixed" my issue of stealing focus from the first selected menu item and therefore not announcing the first selected menu item.. but concerned it may break another scenario I am not testing. 

The problem I did encounter by removing this was that if a user hits `alt` and then opens a menu with `enter` , we did not select the first item and there for we would need this to focus the pane so that up/down arrows would navigate it. I addressed this by doing what I think is the expectation (for windows) and focusing the first item on menu open. (Confirming with accessibility)

Reply:
@niik Continued this conversation in [added comment](https://github.com/desktop/desktop/pull/16315#issuecomment-1472318476).